src/targets/gpu/CMakeLists.txt

```diff
@@ -122,6 +122,7 @@ add_library(migraphx_gpu
     batch_norm_inference.cpp
     clip.cpp
     code_object_op.cpp
+    compile_ops.cpp
```

Comment:
Should we call it compile_pointwise_ops ?

Reply:
No, this can be used to compile more ops in the future.